code/modules/uplink/uplink_items.dm

```diff
@@ -2380,6 +2445,14 @@ GLOBAL_LIST_INIT(illegal_tech_blacklist, typecacheof(list(
 	limited_stock = 1 //you can't use more than one!
 	restricted_roles = list(JOB_NAME_SHAFTMINER)
 
+/datum/uplink_item/badass/syndiecash/plus
```

Comment:
This doesn't make much sense to be reduced for QM and cargo technician in this way considering how much money you get per TC. 15k is enough to buy montains of guns and basically anything in the cargo store.

Reply:
Would 10K be more appropriate or is this idea simply outlandish?

Reply:
Changed it to be a variant of the assistant toolbox that does more damage the more TC is in it, but this one does more damage the more money is in it. May need balance tweaking and will need a more robust way of counting money.

Reply:
Yeah, i think i should let Bacon speak about it since they have higher knowledge regarding damages and money numbers but from a QM perspective on an average shift if, it keeps the 1000 credits = one damage, they could maximize it in one go, what's the higher damage it can do? and how much money is required?

Reply:
I wanted to include a damage fall-off or a damage-cap but haven't gotten around to implementing it yet.